activerecord/test/cases/calculations_test.rb

```diff
@@ -838,7 +838,7 @@ def test_pluck_replaces_select_clause
   def test_pluck_columns_with_same_name
     expected = [["The First Topic", "The Second Topic of the day"], ["The Third Topic of the day", "The Fourth Topic of the day"]]
     actual = Topic.joins(:replies)
```

Comment:
How about adding `order(:id)`?
It is coincidental that id:1's title and id:3's title are alphabetical order.

```suggestion
    actual = Topic.joins(:replies).order(:id)
```

Reply:
Thanks for the review. Updated. 
I should have referred the test `test_pluck_replaces_select_clause` above which also specifies order using `order(:id)`.